### PR TITLE
all: int8 / int16 support

### PR DESCRIPTION
**Pull request by masseelch**

### CODE REVIEW

File: internal/integration/ogent/ent/schema/all_types.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		// field.JSON("nicknames", []string{}),
+		// field.JSON("json_slice", []http.Dir{}).
+		// 	Annotations(entoas.Schema(ogen.String().AsArray())),
+		// field.JSON("json_obj", url.URL{}).
+		// 	Annotations(entoas.Schema(ogen.String())),
+		// field.Other("other", &Link{}).
+		// 	SchemaType(map[string]string{dialect.Postgres: "varchar"}).
+		// 	Default(DefaultLink()).
+		// 	Annotations(entoas.Schema(ogen.String())),
+	}
```
Comment (a8m) [inline internal/integration/ogent/ent/schema/all_types.go:53]: ?

File: template/create.tmpl [hunk 2]
```diff
@@ -5,7 +5,7 @@
 		{{- $a := fieldAnnotation $f }}
 		{{- if or (not $a) (not $a.ReadOnly) }}
 			{{- if $f.Optional }}
-				{{- if $f.IsBytes }}
+				{{- if or $f.IsBytes }}
```
Comment (a8m) [inline template/create.tmpl:8]:
```suggestion
				{{- if $f.IsBytes }}
```

File: internal/integration/ogent/ent/schema/all_types.go (hunk 1 above)
Comment (masseelch) [reply internal/integration/ogent/ent/schema/all_types.go:53]: Reminder to add support for them as well.

**Review by a8m — APPROVED** [review]

File: internal/integration/ogent/ent/schema/all_types.go (hunk 1 above)
Comment (a8m) [reply internal/integration/ogent/ent/schema/all_types.go:53]: I don't like reminders in the code tbh, but it's your project :)

**Pull request merged**